rhoni-team/AppCI
# 7.pylint and autopep config

Close #7 

In this PR were added pylint and autopep for correcting style and python code conventions in backend.

**How to test it?**

1) Install the new dependencies:
`pip install -r requirements.txt`

2) go to the file AppCI -> urls.py and delete the final empty line. Save the file.

![image](https://github.com/rhoni-team/AppCI/assets/39421618/c8eb4314-7cdd-4a51-b003-89bd0ecda2cf)


3) at the same level of requirements.txt file you'll see a Makefile. Go to this level in ubuntu terminal (with the environment activated) and run lint:

```
$ source env/bin/activate
$ make lint
```

You must see a pylint error.

4) fix the error by using autopep:

`$ make autopep`

run lint again to asses that it is working

`$ make lint`
